lib/std/os/linux.zig

```diff
@@ -2610,6 +2610,39 @@ pub fn map_shadow_stack(addr: u64, size: u64, flags: u32) usize {
     return syscall3(.map_shadow_stack, addr, size, flags);
 }
 
+pub const sysinfo_t = extern struct {
```

Comment:
```suggestion
pub const sysinfo = extern struct {
```

Per the UAPI header.

Reply:
I did that just to de-conflict the struct from the helper fn.

Should I change the fn to getSysinfo or maybe; Sysinfo{} sysinfo(*Sysinfo)?

Reply:
Oh right. Usually we uppercase the type name in these cases, so `Sysinfo`.